tests/grid/grid_transform_02.cc

```diff
@@ -83,7 +83,7 @@ int main ()
             }
 
   // then compute new point locations and output the result
-  GridTools::laplace_transform  (new_points,tria);
+  GridTools::laplace_transform  (new_points,tria, static_cast<const Function<dim, double> *>(NULL), true);
```

Comment:
No, this is because NULL is not a function pointer but basically a (long) int and so template deduction doesn't kick in. Learned something new today. :-)